lib/presentation/ui/utils/occurrence/occurrence_small_info.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:vost/constants.dart' as Constants;
+//import 'package:vost/presentation/assets/dimensions.dart';
+import 'package:vost/presentation/assets/text_styles.dart';
+import 'package:vost/presentation/utils/misc.dart';
+
+//import 'occurrence_widget_background.dart';
+
+class OccurrenceSmallInfoWidget extends StatelessWidget {
+  final String _updatedAt;
+  final String _typeName;
+
+  OccurrenceSmallInfoWidget(this._updatedAt, this._typeName);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
```

Comment:
If this is inside the same card, consider using the `OccurrenceWidgetBackground` for consinstency